BitmovinConvivaAnalytics.podspec

```diff
@@ -1,6 +1,6 @@
 Pod::Spec.new do |s|
   s.name             = 'BitmovinConvivaAnalytics'
-  s.version          = '2.0.1'
+  s.version          = '2.0.2'
```

Comment:
Usually, we do set the version when we do the final release. This happens when on develop / a release branch. Please don't update the version in this PR 🙂